services/wordpress/wordpress-version.service.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+
+const BaseWordpress = require('./wordpress-base')
+const { addv } = require('../../lib/text-formatters')
+const { version: versionColor } = require('../../lib/color-formatters')
+
+function VersionForExtensionType(extensionType) {
+  const { capt, exampleSlug } = {
+    plugin: {
+      capt: 'Plugin',
+      exampleSlug: 'bbpress',
+    },
+    theme: {
+      capt: 'Theme',
+      exampleSlug: 'twentyseventeen',
+    },
+  }[extensionType]
+
+  return class WordpressPluginVersion extends BaseWordpress {
+    static get extensionType() {
+      return extensionType
+    }
+
+    static render({ response }) {
+      return {
+        message: addv(response.version),
+        color: versionColor(response.version),
+      }
+    }
+
+    static get category() {
+      return 'version'
+    }
+
+    static get defaultBadgeData() {
+      return { label: 'version' }
```

Comment:
Can we make the label on these ones `extensionType()` rather than version. The existing plugin version badge uses 'plugin': ![](https://img.shields.io/wordpress/plugin/v/akismet.svg) https://img.shields.io/wordpress/plugin/v/akismet.svg and we tend to use the platform/package type as the label on version badges